internal/sync/tar.go

```diff
@@ -84,25 +84,38 @@ func (t *Tar) Sync(ctx context.Context, service string, paths []*PathMapping) er
 	if len(pathsToDelete) != 0 {
 		deleteCmd = append([]string{"rm", "-rf"}, pathsToDelete...)
 	}
-	var eg multierror.Group
+
+	var (
+		eg    errgroup.Group
+		errMu sync.Mutex
+		errs  = make([]error, 0, len(containers)*2) // max 2 errs per container
+	)
+
+	eg.SetLimit(16) // arbitrary limit, adjust to taste :D
```

Comment:
For this one, I used an `errgroup` so that we have the option to limit concurrency, but we could use a regular waitgroup as well